Game/Game.Test.js

```diff
@@ -1,178 +1,166 @@
-const chai = require("chai");
-const expect = chai.expect;
-
 let Game = require("./Game");
+const Player = require('../Player/Player.js');
 
 describe('Game', () => {
+
   let a;
   beforeEach(() => {
-    a = new Game(3);
-    a.setWordToGuess('game')
+    a = new Game();
+    a.setGuessWord('game')
```

Comment:
Maybe 'game' should be the new Game and something like "word" or "guess" should be the setGuessWord? Just thinking about dev empathy